CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - We fixed an issue where groups generated from authors' last names did not include all entries of the authors' [#5833](https://github.com/JabRef/jabref/issues/5833)
 - The export to MS Office XML now uses the month name for the field `MonthAcessed` instead of the two digit number [#7354](https://github.com/JabRef/jabref/issues/7354)
 - We included some standalone dialogs from the options menu in the main preference dialog and fixed some visual issues in the preferences dialog. [#7384](https://github.com/JabRef/jabref/pull/7384)
+- Readd query syntax validation. [#7411](https://github.com/JabRef/jabref/pull/7411)
```

Comment:
I'm just thinking: If remove a feature and readd it fixed in the same release cycle, we probably should move this changelog entry to the fixed section: We fixed an issue with search freezing jabref or so...

Reply:
Done!